packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js

```diff
@@ -21,6 +21,12 @@ export default /* @ngInject */ ($stateProvider) => {
         Object.values(DASHBOARD_FEATURES).filter((feature) =>
           capabilities.includes(feature),
         ),
+      featureFlipping: /* @ngInject */ (ovhFeatureFlipping) => {
+        const features = `vps:secondary-dns`;
+        return ovhFeatureFlipping.checkFeatureAvailability(features);
+      },
```

Comment:
```suggestion
      featureFlipping: /* @ngInject */ (ovhFeatureFlipping) => ovhFeatureFlipping.checkFeatureAvailability('vps:secondary-dns'),
```

Reply:
And see just above "features" with DASHBOARD_FEATURES and capabilities maybe could we reuse it ?